hooks/presence.py

```diff
@@ -53,19 +53,18 @@ def __init__(self, *args, **kwargs):
             'txt': None
         }
 
-    def present(self, engine, start=None, details="Regretting...", state=None):
+    def present(self, engine, start=None, details="Regretting...", state=None, url=None, thumb="icon"):
         """
         Set status for DiscordRPC.
         """
         self._details = {
             'details': details,
             'state': state,
             'start': time.time()*1000 - start if start else None,
+            'thumb': thumb,
+            'buttons': [ { "label": "View Anime", "url": url } ] if url else None,
```

Comment:
you should use media type here, instead of `"View Anime"`

Reply:
Can you expound on what you are trying to suggest?

Reply:
Ah, you mean something like "Anilist" or "MyAnimeList" as the label?

Reply:
`"View %s" % engine.api_info['mediatype']` something like this... but `show['type']` would've been better.

or instead of just passing url, you can pass a button array directly.

Reply:
`"View %s" % engine.api_info['mediatype']` would do, something like `View TV` or just `TV` seems weird for `show['type']`.